app/models/miq_provision.rb

```diff
@@ -97,6 +97,10 @@ def after_ae_delivery(ae_result)
     end
   end
 
+  def dialog_values
+    options
+  end
```

Comment:
This is the only thing that's weird to me - I feel like this method needs a rename or something, because these aren't dialog value anymore, they are all the values.